content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
++++
+authors = "Arohi Parikh"
+categories = ["Careers", "Software Craftsmanship"]
+date = 2021-09-21T00:00:00Z
```

Comment:
We should change this date so that it now comes in the latest blogs

Reply:
Done